src/packer.pkr.hcl

```
@@ -83,51 +102,48 @@ source "amazon-ebs" "example" {
   region             = var.build_region
   region_kms_key_ids = var.region_kms_keys
   skip_create_ami    = var.skip_create_ami
-  source_ami         = data.amazon-ami.debian_bullseye.id
-  ssh_username       = "admin"
+  source_ami         = data.amazon-ami.windows_server_2022.id
   subnet_filter {
     filters = {
       "tag:Name" = "AMI Build"
     }
   }
   tags = {
-    Application        = "Example"
-    Base_AMI_Name      = data.amazon-ami.debian_bullseye.name
+    Application        = "Windows Server 2022"
+    Base_AMI_Name      = data.amazon-ami.windows_server_2022.name
     GitHub_Release_URL = var.release_url
-    OS_Version         = "Debian Bullseye"
+    OS_Version         = "Windows Server 2022"
     Pre_Release        = var.is_prerelease
     Release            = var.release_tag
     Team               = "VM Fusion - Development"
   }
   # Many Linux distributions are now disallowing the use of RSA keys,
   # so it makes sense to use an ED25519 key instead.
   temporary_key_pair_type = "ed25519"
+  user_data_file          = "src/winrm_bootstrap.txt"
```

This is an ssh specific setting and since we are using WinRM it can be removed.

```suggestion
  user_data_file = "src/winrm_bootstrap.txt"
```

---

@mcdonnnj i've previously removed this in a previous comment roughly a couple months ago and was told to add it back to keep it consistent with the underlying skeleton. I can have it removed if you'd like but though i'd let you know.

---

@mcdonnnj resolved with [d8d642e](https://github.com/cisagov/windows-server-packer/pull/10/commits/d8d642eaa7d1c98b9b54cb5c526e4c7154f39480)

---

@dav3r @jsf9k I want to make sure this is a team agreed on decision. I asked for this option to be removed because we are using the [`winrm` communicator](https://www.packer.io/docs/communicators/winrm) and this is an option for the [`ssh` communicator](https://www.packer.io/docs/communicators/ssh). Even if it does not impact functionality I do not think we should include potentially misleading options if they are unused for the current configuration.

---

I agree with your logic @mcdonnnj.  If it's not needed here, get rid of it.

---

@mcdonnnj @dav3r i already committed the work to have it removed. this should be considered resolved

---

@itsmostafa I just wanted confirmation from the rest of my team that we were all on board based on the comment you left before adding a resolving commit.

---

I agree @mcdonnnj.